### ISSUE TITLE
flow do not output RSSPEC

### ISSUE THREAD

**Issue by hnil** [OPM/opm-simulators#1063]
When runnining flow UNRST is written but not RSSPEC which is at  convenient for reading UNRST.

**Comment by joakim-hove** [comment]
No - that is true, and the binary readers in ert can not read them either. Have looked through this several times and concluded that this file does not really give much.

If you describe what you want to achieve we might be able to help you.

**Comment by hnil** [comment]
I had some scripts reading eclipse output (MRST), which use this file for indexing. 
The reason we use this (earlier version did not) is that it makes reading of spesific fields much faster in particular on large models. If RSPECS is provided on can jump directly to the data in the output file.

**Comment by hnil** [comment]
Other small difference noticed between eclipse output and OPM is that INIT file do not contain NTG...

**Comment by alfbr** [comment]
> Other small difference noticed between eclipse output and OPM is that INIT file do not contain NTG...

That landed today though:
https://github.com/OPM/opm-output/pull/167

**Comment by andlaus** [comment]
are there any news here?

**Comment by atgeirr** [comment]
No, we still do not output an RSSPEC file, so this issue should remain open.

**Comment by sermk** [comment]
Neither it creates INSPEC. Would like to second the request for index files output!

**Comment by OPMUSER** [comment]
We still need to add INSPEC and RSSPEC unformatted output index files as well as the equivalent formatted output files (FINSPEC and FRSSPEC) for compatibility with the commercial simulator and third party post processing software.

**Comment by bska** [comment]
> any plan to fix this soon ?

To be honest, we don't currently have any plans to add these files, mainly because [ResInsight](https://resinsight.org/) does not need them&ndash;or even use them if they exist.  As with anything, it's a matter of priorities and we're a fairly small team of developers.
